__tests__/app.test.js

```diff
@@ -282,3 +280,31 @@ test(" response with 204 and deletes the comment", () => {
       });
     });
 });
+
+describe("users endpoint ", () => {
+  test.only("response with the status of 200 and array of all the available users ", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body }) => {
+        expect(body.users).toBeInstanceOf(Array);
+        expect(body.users.length).toBeGreaterThan(0);
+
+        body.users.forEach((user) => {
+          expect(user).toMatchObject({
+            username: expect.any(String),
+            name: expect.any(String),
+            avatar_url: expect.any(String),
+          });
+        });
+      });
+  });
+  test("response with the status of 404 and an appropriate error message if wrong endpoint name is used ", () => {
+    return request(app)
+      .get("/api/userssss")
+      .expect(404)
+      .then((res) => {
+        expect(res.body.message).toBe("Not Found");
+      });
+  });
```

Comment:
This is not a valid test for this endpoint this is a generic 404 error handler which is the same as /api/banana and should not be in the describe block for users endpoint. I also cant be sure this test is passing because of the .only on the previous test